styles/elements/inputs/checkbox/_checkbox.scss

```diff
@@ -0,0 +1,95 @@
+// Checkbox
+//
+// Checkbox styling to use throughout the app. This class can be applied
+// directly to checkbox inputs, or can be applied to parent elements that
+// contain a checkbox. The later is used for compatibility with IE11.
+//
+// Markup:
+// <input type="checkbox" class="checkbox" />
+//
+// <span class="checkbox">
+//   <input type="checkbox" />
+//   <span></span>
```

Comment:
Is the span an optional form label? Or just an IE thing?

Reply:
The extra span is an element that we style using sibling selectors:

```css
input[type="checkbox"]:checked + * {

}
```

I couldn't find any other way to style sibling/child elements that was supported by IE. It doesn't support `:after` or `:before` within `input` elements, and you can't target `:before` or `:after` of a parent element with the sibling selector.

The current implementation uses `*` so it could be a span or any other tag, there just needs to be the extra tag there. I was actually thinking that we could write an HTMLBars transform that polyfills this at build time based on build target, but that's a future thing.